pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -59,7 +59,8 @@ extend type Mutation {
   @hasRole(role: MINT_USER)
   updateMTOMilestone(id: UUID!, changes: MTOMilestoneChanges!): MTOMilestone!
   @hasRole(role: MINT_USER)
-  #TODO (mto) will we need any other mutations here? Can we ever delete?
+  deleteMTOMilestone(id: UUID!): Boolean!
```

Comment:
Question: Should we return a milestone here instead of Boolean? It seems slightly more versatile to me.

Reply:
Added some comments in 53a587fe7e2caeacf386dd75fb0c92cc6cece6a0 -- but leaving this as-is for the sake of time 😁